mongodbatlas/data_source_mongodbatlas_project.go

```diff
@@ -81,6 +81,26 @@ func dataSourceMongoDBAtlasProject() *schema.Resource {
 					},
 				},
 			},
+			"is_collect_database_specifics_statistics_enabled": {
+				Type:     schema.TypeBool,
+				Computed: true,
+			},
```

Comment:
[q] Could you update the tests with these new fields?

Reply:
Tests added

Reply:
Could you show me where? I do not see them

Reply:
Added tests to datasource to verify new attributes exist